codegen/smithy-dafny-codegen/src/main/java/software/amazon/polymorph/smithygo/localservice/DafnyLocalServiceTypeConversionProtocol.java

```diff
@@ -7,6 +7,7 @@
 import java.util.HashSet;
```

Comment:
Let's add Operation and Orphan shapes to the `SimpleErrors` TestModel. Otherwise LGTM.

Reply:
I'm going to hold off on TestModel changes for this PR -- I'm planning to add orphaned error coverage in OrphanedShapes TestModel